components/google_sheets/google_sheets.app.js

```diff
@@ -2,7 +2,8 @@ const { google } = require("googleapis");
 const google_drive = require("../google_drive/google_drive.app");
 
 module.exports = {
-  ...google_drive,
+  type: "app",
+  app: "google_sheets",
```

Comment:
We still want to inherit other properties, and only override the `app` attribute in this case.
This will achieve that:
```suggestion
  ...google_drive,
  app: "google_sheets",
```